pkg/test/auth/tokenmanager.go

```diff
@@ -144,9 +144,29 @@ func (tg *TokenManager) Key(kid string) (*rsa.PrivateKey, error) {
 	return key, nil
 }
 
+/****************************************************
+
+  This section is a temporary fix until formal leeway support is available in the next jwt-go release
+
+ *****************************************************/
+
+const leeway = 5
+
+type MyClaims struct {
+	*jwt.StandardClaims
+}
+
+func (c *MyClaims) Valid() error {
+	c.StandardClaims.ExpiresAt += leeway
+	err := c.StandardClaims.Valid()
+	c.StandardClaims.ExpiresAt -= leeway
+	return err
+}
+
 // GenerateToken generates a default token.
 func (tg *TokenManager) GenerateToken(identity Identity, kid string, extraClaims ...ExtraClaim) *jwt.Token {
 	token := jwt.New(jwt.SigningMethodRS256)
+	token.Claims = &MyClaims{&jwt.StandardClaims{ExpiresAt: time.Now().Unix() - 1}}
```

Comment:
i think this is the same as the "exp" field which is specified down below: https://github.com/codeready-toolchain/toolchain-common/pull/63/files#diff-c2bd6592ef1ff7e697a51858a44abfaeR176

Reply:
Good catch, yes it is the same claim.  I'm wondering now though if the `ExpiresAt` claim is the actual problem here.  Isn't the problem that the token is being issued at a time that appears to the parser to be in the future?  If that is the case, wouldn't it really be the `iat` (issued at) claim that we need to apply the leeway to?  I've been working off the basis of [this comment](https://github.com/dgrijalva/jwt-go/issues/257#issuecomment-494597343) and made the changes accordingly, however on further reflection this doesn't really make sense now in the context of our issue.  What do you think?

Reply:
Yeah, so when i was attempting to fix this problem changing the `exp` field didn't seem to have any impact on fixing it. It was more the `iat` field - i had set it back by 60 seconds and that seemed to solve it for me. So i think this code contains the right idea, it just needs to do it for `iat` instead of `exp.`

Reply:
That makes way more sense to me too.  I'll keep working on trying to fix the StandardClaims / MapClaims incompatibility and make the fix to iat instead of exp.